test-e2e/conftest.py

```diff
@@ -0,0 +1,4 @@
+# Hack to exclude test-e2e/conftest.py from the top-level tox config py35-unittests.
+# https://stackoverflow.com/a/37493203
+pytest_plugins = ['test_e2e_module']
```

Comment:
Cool! I hope later we can figure out a better way to organise the DC/OS repository but that isn't my #1 concern right now.